visual-testing-app/src/components/info-main-page/info-main-page.visualroute.tsx

```diff
@@ -0,0 +1,35 @@
+import { InfoMainPage } from '@commercetools-frontend/application-components';
+import TextField from '@commercetools-uikit/text-field';
+import Spacings from '@commercetools-uikit/spacings';
+import { Suite, Spec } from '../../test-utils';
+
+export const routePath = '/info-main-page';
+
+const Content = () => (
+  <Spacings.Stack scale="l">
+    <Spacings.Inline scale="l">
+      <TextField title="First Name" value="foo" />
+      <TextField title="Last Name" value="foo" />
+    </Spacings.Inline>
+    <TextField title="Email Address" value="foo" />
+    <TextField title="Business Role" value="foo" />
```

Comment:
I guess we need to add that, otherwise it throws in dev mode
```suggestion
      <TextField title="First Name" value="foo" onChange={() => { }} />
      <TextField title="Last Name" value="foo" onChange={() => { }} />
    </Spacings.Inline>
    <TextField title="Email Address" value="foo" onChange={() => { }} />
    <TextField title="Business Role" value="foo" onChange={() => { }} />
```

Reply:
[d08f505](https://github.com/commercetools/merchant-center-application-kit/pull/2745/commits/d08f5056b47a724ec8eb97936f47ce9bdc11683a)